distribution/docs/src/main/jdocs/content/_security-framework/security-encryption-commands-contents.adoc

```diff
@@ -0,0 +1,31 @@
+:title: Security Encryption Commands
+:type: subSecurityFramework
+:status: published
+:parent: Security Encryption
+:order: 02
+:summary: Security Encryption commands.
+
+The Security Encryption Commands bundle enhances the ${branding} system console by allowing administrators and integrators to encrypt and decrypt values directly from the console.
+
+The `security:encrypt` command allows plain text to be encrypted.
```

Comment:
Should probably state that this uses HMAC + AES for encryption with a randomly generated key that is created when the system is installed.

Reply:
added.